app/controllers/maintainers_controller.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+class MaintainersController < ApplicationController
+  before_action :load_project
+  def edit
+  end
+
+  def create
+    user = User.find_by(id: params[:maintainer_id])
+    @project.maintainers << user
```

Comment:
Nekem itt hiányzik az a gondolat, hogy "Mi van akkor, ha nem sikerült létrehozni?".

Valami hasonló jobban tetszene:

```ruby
def create
  resource = authorize Resource.find_by(id: params[:id])
  if collection << resource
    flash[:info] = 'nice job!'
    redirect_to appropriate_path
  else
    flash[:danger] = 'uh-oh'
    redirect_to appropriate_path_on_failre_path
  end
end
```

Reply:
A fenti példában:
- visszajelzünk a felhasználó felé, ha sikertelen az akció
- lekezeljük, ha sikertelen az akció
- authorizáljuk a kapcsolat másik oldalát is (nálad a user erőforrás)

Visszajelzésre nem mindig a statikus "Something went wrong!" a legjobb megoldás, amit fent csináltam. Érdemes lehet kipróbálnod a következő függvényeket:

```ruby
resource.errors
resource.errors.full_messages
resource.errors.full_messages.to_sentence
```